content/en/internal/configuration-database/service-info/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Service information"
+linkTitle: "Service info"
+weight: 100
+type: "docs"
+draft: false
+description: >
+    Technical details of the configuration database
+---
+
+## Identity card
+
+<!-- markdownlint-disable line-length no-bare-urls -->
+
+| Property                     | Value                                                                                               |
+| ---------------------------- | ---------------------------------------------------------------------------------|
```

Comment:
Is the header end marker correctly aligned (or is it only due the GitHub rendering?)?